doc/source/getting_started/user_guide.rst

```diff
@@ -1,33 +1,30 @@
-===============
-Getting started
-===============
+==========
+User guide
+==========
 
-To run the Antenna Toolkit, you must have a licensed copy of AEDT installed.
 You have multiple options for installing and launching this toolkit:
 
-- You can install the toolkit directly in AEDT via an installation script and then launch it
-  as a wizard. For more information, see :ref:`install-toolkit-AEDT`.
-- You can install the toolkit from the AEDT console and then launch it as a wizard. For more
-  information, see :ref:`install_toolkit_console_ui`.
-- You can install and launch the toolkit directly from a Python console and then use the toolkit's APIs.
+- You can install the toolkit directly in AEDT via an installation script and then launch it as a wizard.
+  For more information, see :ref:`install-toolkit-AEDT`.
+- You can install the toolkit from a Python console and then launch Antenna wizard.
```

Comment:
```suggestion
- You can install the toolkit from a Python console and then launch the Antenna wizard.
```
If wizard is capitalized in the UI's title bar, then it should always be capitalized as a product name here.